--- build.sho	2024-05-28 04:03:10.371282170 +0000
+++ build.sh	2024-05-28 04:05:48.457003408 +0000
@@ -26,3 +26,11 @@
 # Copy out dictionary
 cp $SRC/liblouis/tests/fuzzing/fuzz_translate_generic.dict $OUT/fuzz_translate_generic.dict
 cp $SRC/liblouis/tests/fuzzing/fuzz_translate_generic.dict $OUT/fuzz_backtranslate.dict
+
+
+$CC $CFLAGS -c fuzz_newfuzzer.c -o fuzz_newfuzzer.o \
+    -I$SRC/liblouis -I$SRC/liblouis/liblouis
+$CXX $CXXFLAGS $LIB_FUZZING_ENGINE fuzz_newfuzzer.o \
+    -o $OUT/fuzz_newfuzzer fuzz_lib.a
+
+cp $SRC/liblouis/tests/fuzzing/fuzz_translate_generic.dict $OUT/fuzz_newfuzzer.dict
