--- Makefileo	2023-09-17 21:15:33.031639037 +0000
+++ Makefile	2023-09-17 21:16:49.204026298 +0000
@@ -33,7 +33,8 @@
 PHP_FUZZER_UNSERIALIZEHASH_OBJS = sapi/fuzzer/fuzzer-unserializehash.lo sapi/fuzzer/fuzzer-sapi.lo
 PHP_FUZZER_JSON_OBJS = sapi/fuzzer/fuzzer-json.lo sapi/fuzzer/fuzzer-sapi.lo
 PHP_FUZZER_EXIF_OBJS = sapi/fuzzer/fuzzer-exif.lo sapi/fuzzer/fuzzer-sapi.lo
-PHP_FUZZER_BINARIES = sapi/fuzzer/php-fuzz-parser sapi/fuzzer/php-fuzz-execute sapi/fuzzer/php-fuzz-function-jit sapi/fuzzer/php-fuzz-tracing-jit sapi/fuzzer/php-fuzz-unserialize sapi/fuzzer/php-fuzz-unserializehash sapi/fuzzer/php-fuzz-json sapi/fuzzer/php-fuzz-exif
+PHP_FUZZER_NEW_OBJS = sapi/fuzzer/fuzzer-exif.lo sapi/fuzzer/fuzzer-sapi.lo
+PHP_FUZZER_BINARIES = sapi/fuzzer/php-fuzz-parser sapi/fuzzer/php-fuzz-execute sapi/fuzzer/php-fuzz-function-jit sapi/fuzzer/php-fuzz-tracing-jit sapi/fuzzer/php-fuzz-unserialize sapi/fuzzer/php-fuzz-unserializehash sapi/fuzzer/php-fuzz-json sapi/fuzzer/php-fuzz-exif sapi/fuzzer/php-fuzz-new
 PROG_SENDMAIL = /usr/sbin/sendmail
 DASM_FLAGS = -D X64=1
 DASM_ARCH = x86
