--- build.sh.o	2023-09-12 02:14:13.193109818 +0000
+++ build.sh	2023-09-12 02:16:30.445778296 +0000
@@ -39,3 +39,22 @@
 
 # Build seed corpus
 zip $OUT/xml_parser_fuzzer_seed_corpus.zip $SRC/fuzzing-corpus/xml/test.xml
+
+$CXX $CXXFLAGS -I../../Source \
+        -I../../build-dir/Source \
+        -c new_fuzzer.cc \
+        -o new_fuzzer.o
+
+
+export cmexpat_dir="${SRC}/CMake/build-dir/Utilities/cmexpat/CMakeFiles/cmexpat.dir/lib"
+$CXX $CXXFLAGS $LIB_FUZZING_ENGINE \
+                new_fuzzer.o -o $OUT/new_fuzzer \
+                ../../build-dir/Source/CMakeFiles/CMakeLib.dir/cmXMLParser.cxx.o \
+                $cmexpat_dir/xmlparse.c.o \
+                $cmexpat_dir/xmlrole.c.o \
+                $cmexpat_dir/xmltok.c.o
+
+
+# Build seed corpus
+zip $OUT/new_fuzzer_seed_corpus.zip $SRC/fuzzing-corpus/xml/test.xml
+
