--- build.sho	2023-09-17 16:59:44.155099906 +0000
+++ build.sh	2023-09-17 17:01:16.167356808 +0000
@@ -21,3 +21,7 @@
 cd src
 $CC $CFLAGS -c $SRC/fuzz_burl.c -I. -I../include
 $CXX $CXXFLAGS $LIB_FUZZING_ENGINE fuzz_burl.o lighttpd-burl.o lighttpd-buffer.o lighttpd-base64.o lighttpd-ck.o -o $OUT/fuzz_burl
+
+$CC $CFLAGS -c $SRC/fuzz_new.c -I. -I../include
+$CXX $CXXFLAGS $LIB_FUZZING_ENGINE fuzz_new.o lighttpd-burl.o lighttpd-buffer.o lighttpd-base64.o lighttpd-ck.o -o $OUT/fuzz_new
+
