--- Makefile.amo	2023-09-12 22:29:33.933525960 +0000
+++ Makefile.am	2023-09-12 22:30:54.149900334 +0000
@@ -28,7 +28,8 @@
 EXTRA_PROGRAMS=\
 	fuzzer-rules \
 	fuzzer-uevent \
-	fuzzer-usb-descriptor
+	fuzzer-usb-descriptor \
+        fuzzer-new
 
 fuzzer_rules_SOURCES=\
 	fuzzer-rules.cpp
@@ -38,3 +39,7 @@
 
 fuzzer_usb_descriptor_SOURCES=\
 	fuzzer-usb-descriptor.cpp
+
+fuzzer_new_SOURCES=\
+        fuzzer-new.cpp
+
