--- Makefile	2023-09-19 04:51:08.493439762 +0000
+++ Makefile	2023-09-19 04:52:05.661669911 +0000
@@ -9,11 +9,14 @@
 INCLUDE_DIR=-I ${CPPCHECK_DIR}/lib -I ${CPPCHECK_DIR}/externals/picojson -I ${CPPCHECK_DIR}/externals/simplecpp -I ${CPPCHECK_DIR}/externals/tinyxml2 -I ${CPPCHECK_DIR}/externals
 SRC_FILES=main.cpp type2.cpp ${CPPCHECK_DIR}/externals/simplecpp/simplecpp.cpp ${CPPCHECK_DIR}/externals/tinyxml2/tinyxml2.cpp ${CPPCHECK_DIR}/lib/*.cpp
 
-all:	oss-fuzz-client	translate
+all:	oss-fuzz-client	translate newfuzzer
 
 oss-fuzz-client:	main.cpp	type2.cpp	type2.h
 	${CXX} -std=c++11 -g ${CXXFLAGS} -o oss-fuzz-client ${INCLUDE_DIR} ${SRC_FILES} ${LIB_FUZZING_ENGINE}
 
+newfuzzer:	newfuzzer.cpp	type2.cpp	type2.h
+	${CXX} -std=c++11 -g ${CXXFLAGS} -o newfuzzer ${INCLUDE_DIR} ${SRC_FILES} ${LIB_FUZZING_ENGINE}
+
 translate:	translate.cpp	type2.cpp	type2.h
 	${CXX} -std=c++11 -g ${CXXFLAGS} -o translate type2.cpp translate.cpp
 
