--- build.sho	2025-10-13 16:38:05.870969531 +0000
+++ build.sh	2025-10-13 16:45:58.424482806 +0000
@@ -8,7 +8,7 @@
 # It may be necessary to leave CMAKE_BUILD_TYPE empty and specify LIBXML2_WITH_MODULES=OFF
 # in order for fuzz introspector builds to succeed (details unverified).
 cd $SRC
-wget https://download.gnome.org/sources/libxml2/2.15/libxml2-2.15.0.tar.xz
+#wget https://download.gnome.org/sources/libxml2/2.15/libxml2-2.15.0.tar.xz
 tar xf libxml2-2.15.0.tar.xz
 cd libxml2-2.15.0
 mkdir build && cd build
@@ -93,10 +93,10 @@
 # disabled for UBSan:
 #  - read_dimacs_flow, needs a complete rewrite, see https://github.com/igraph/igraph/issues/1924
 #  - write_all_gml|graphml, uses `(igraph_int_t) x == x` to check representability as integer; this triggers UBSan
-TARGETS="read_edgelist read_dl read_gml read_graphdb read_graphml read_lgl read_ncol read_pajek bliss edge_connectivity vertex_connectivity vertex_separators basic_properties_directed basic_properties_undirected linear_algos_directed linear_algos_undirected centrality community weighted_centrality weighted_community misc_algos misc_algos_weighted spatial"
+TARGETS="read_edgelist read_dl read_gml read_graphdb read_graphml read_lgl read_ncol read_pajek bliss edge_connectivity vertex_connectivity vertex_separators basic_properties_directed basic_properties_undirected linear_algos_directed linear_algos_undirected centrality community weighted_centrality weighted_community misc_algos misc_algos_weighted spatial newfuzzer"
 if [ "$SANITIZER" != "undefined" ]
 then
-  TARGETS="$TARGETS read_dimacs_flow write_all_gml write_all_graphml"
+  TARGETS="$TARGETS read_dimacs_flow write_all_gml write_all_graphml newfuzzer"
 fi
 
 for TARGET in $TARGETS
