--- build.sho	2025-10-09 16:38:33.763438127 +0000
+++ build.sh	2025-10-09 16:39:45.289556172 +0000
@@ -17,7 +17,7 @@
 
 # Get data. We do this here to resolve CIFuzz issues
 # CC https://github.com/uclouvain/openjpeg/pull/1386
-git clone --depth 1 https://github.com/uclouvain/openjpeg-data data
+#git clone --depth 1 https://github.com/uclouvain/openjpeg-data data
 export OPJ_DATA_ROOT=$PWD/data
 
 mkdir build
