--- Makefile.o	2023-09-16 01:29:26.680362615 +0000
+++ Makefile	2023-09-16 01:36:12.326551486 +0000
@@ -3,7 +3,8 @@
 
 FUZZERS:= \
 	broker_fuzz_read_handle \
-	broker_fuzz_test_config
+	broker_fuzz_test_config \
+	new_fuzzer
 
 CPPFLAGS+=-I${R}/include/ -I${R}/src -I${R}/lib -I${R} -I${R}/common -I${R}/deps \
 	-DWITH_BRIDGE -DWITH_BROKER -DWITH_CONTROL -DWITH_EC -DWITH_EPOLL \
@@ -19,6 +20,11 @@
 	$(CXX) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) -o $@ $< $(LIBADD)
 	install $@ ${OUT}/$@
 	cp ${R}/fuzzing/corpora/broker_packet_seed_corpus.zip ${OUT}/$@_seed_corpus.zip
+
+new_fuzzer : new_fuzzer.cpp ${R}/src/mosquitto_broker.a
+	$(CXX) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) -o $@ $< $(LIBADD)
+	install $@ ${OUT}/$@
+	cp ${R}/fuzzing/corpora/broker_packet_seed_corpus.zip ${OUT}/$@_seed_corpus.zip
 
 broker_fuzz_test_config : broker_fuzz_test_config.cpp ${R}/src/mosquitto_broker.a
 	$(CXX) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) -o $@ $< $(LIBADD)
