--- build.sho	2025-02-19 02:59:28.457846565 +0000
+++ build.sh	2025-02-19 03:02:41.092865762 +0000
@@ -106,9 +106,9 @@
 # https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=45633
 # To make sure all the fuzz targets use the same version of zlib
 # it's also built with ASan and UBSan.
-git clone https://github.com/madler/zlib
+#git clone https://github.com/madler/zlib
 pushd zlib
-git checkout v1.3.1
+#git checkout v1.3.1
 if ! ./configure --static; then
     cat configure.log
     exit 1
@@ -155,3 +155,13 @@
 	./libasm/libasm.a ./libebl/libebl.a ./backends/libebl_backends.a ./libcpu/libcpu.a \
   ./libdw/libdw.a ./libelf/libelf.a ./lib/libeu.a "$zlib" \
 	-o "$OUT/fuzz-libdwfl"
+
+$CC $CFLAGS \
+  -D_GNU_SOURCE -DHAVE_CONFIG_H \
+  -I. -I./lib -I./libelf -I./libebl -I./libdw -I./libdwelf -I./libdwfl -I./libasm \
+  -c "$SRC/fuzz-newfuzz.c" -o fuzz-newfuzz.o
+$CXX $CXXFLAGS $LIB_FUZZING_ENGINE fuzz-newfuzz.o \
+        ./libasm/libasm.a ./libebl/libebl.a ./backends/libebl_backends.a ./libcpu/libcpu.a \
+  ./libdw/libdw.a ./libelf/libelf.a ./lib/libeu.a "$zlib" \
+        -o "$OUT/fuzz-newfuzz"
+
