--- make_fuzzers.o	2023-09-12 21:55:23.375853035 +0000
+++ make_fuzzers	2023-09-12 21:57:49.444516861 +0000
@@ -36,4 +36,25 @@
    	 \$(LIB_FUZZING_ENGINE) $ngx_libs$ngx_link$ngx_main_link -lcrypt
 $ngx_long_end					    
 
+
+fuzzers: objs/new_fuzzer
+
+objs/src/fuzz/wrappers.o:
+        \$(CC) $ngx_compile_opt \$(CFLAGS) -o objs/src/fuzz/wrappers.o src/fuzz/wrappers.c
+
+objs/new_fuzzer: $ngx_deps_fuzz
+         \$(CXX) \$(CXXFLAGS) -DNDEBUG src/fuzz/new_fuzzer.cc \
+         src/fuzz/genfiles/http_request_proto.pb.cc \
+         -o objs/http_request_fuzzer \
+         \$(CORE_INCS) \$(HTTP_INCS) \
+         -I src/fuzz/genfiles/ -I \$(SRC)/ \
+         -I \$(SRC)/libprotobuf-mutator/ \
+         -I \$(SRC)/LPM/external.protobuf/include \
+         \$(SRC)/LPM/src/libfuzzer/libprotobuf-mutator-libfuzzer.a \
+         \$(SRC)/LPM/src/libprotobuf-mutator.a \
+         -Wl,--start-group \$(SRC)/LPM/external.protobuf/lib/lib*.a -Wl,--end-group \
+         $ngx_binexit$ngx_long_cont$ngx_objs \
+         \$(LIB_FUZZING_ENGINE) $ngx_libs$ngx_link$ngx_main_link -lcrypt
+$ngx_long_end
+
 END
