--- build.sho	2024-06-03 00:04:05.968174684 +0000
+++ build.sh	2024-06-03 00:07:56.972677729 +0000
@@ -16,14 +16,14 @@
 ################################################################################
 
 # build project
-export LDFLAGS="$CXXFLAGS"
+export LDFLAGS="$CXXFLAGS -D_FILE_OFFSET_BITS=64"
 $SRC/e2fsprogs/configure
 make -j$(nproc) all
 
 # build fuzzers
 for fuzzer in $(find $SRC/fuzz -name '*_fuzzer.cc'); do
   fuzzer_basename=$(basename -s .cc $fuzzer)
-  $CXX $CXXFLAGS \
+  $CXX $CXXFLAGS -D_FILE_OFFSET_BITS=64 \
       $LIB_FUZZING_ENGINE \
       -I $SRC/e2fsprogs/lib \
       $fuzzer \
