--- Makefile.o	2024-06-03 00:13:25.604520729 +0000
+++ Makefile	2024-06-03 00:14:19.754361372 +0000
@@ -99,7 +99,7 @@
 BUILD_CC = clang
 PTHREAD_CFLAGS = -pthread
 PTHREAD_LIBS = -lpthread
-CFLAGS = -O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION  -fsanitize=fuzzer-no-link
+CFLAGS = -O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION  -fsanitize=fuzzer-no-link -D_FILE_OFFSET_BITS=64
 CFLAGS_SHLIB = -O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION  -fsanitize=fuzzer-no-link
 CFLAGS_STLIB = -O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION  -fsanitize=fuzzer-no-link
 CPPFLAGS = -I. -I$(top_builddir)/lib -I$(top_srcdir)/lib
