--- build.sh	2024-06-03 18:42:00.680504319 +0000
+++ build.sho	2024-06-03 18:39:50.659727292 +0000
@@ -93,10 +93,10 @@
 # disabled for UBSan:
 #  - read_dimacs_flow, needs a complete rewrite, see https://github.com/igraph/igraph/issues/1924
 #  - write_all_gml|graphml, uses `(igraph_integer_t) x == x` to check representability as integer; this triggers UBSan
-TARGETS="read_edgelist read_dl read_gml read_graphdb read_graphml read_lgl read_ncol read_pajek edge_connectivity vertex_connectivity vertex_separators basic_properties_directed basic_properties_undirected linear_algos_directed linear_algos_undirected centrality community weighted_centrality weighted_community misc_algos newfuzzer"
+TARGETS="read_edgelist read_dl read_gml read_graphdb read_graphml read_lgl read_ncol read_pajek edge_connectivity vertex_connectivity vertex_separators basic_properties_directed basic_properties_undirected linear_algos_directed linear_algos_undirected centrality community weighted_centrality weighted_community misc_algos"
 if [ "$SANITIZER" != "undefined" ]
 then
-  TARGETS="$TARGETS read_dimacs_flow write_all_gml write_all_graphml newfuzzer"
+  TARGETS="$TARGETS read_dimacs_flow write_all_gml write_all_graphml"
 fi
 
 for TARGET in $TARGETS
