--- build.sh	2023-11-04 01:41:50.718636227 +0000
+++ build.sh	2023-11-04 01:45:25.893350040 +0000
@@ -21,7 +21,7 @@
 export LDFLAGS="-l:libbsd.a"
 
 # Download apr and place in httpd srclib folder. Apr-2.0 includes apr-utils
-svn checkout https://svn.apache.org/repos/asf/apr/apr/trunk/ srclib/apr
+# svn checkout https://svn.apache.org/repos/asf/apr/apr/trunk/ srclib/apr
 
 # Build httpd
 ./buildconf
@@ -31,7 +31,7 @@
 static_pcre=($(find /src/pcre2 -name "libpcre2-8.a"))
 
 # Build the fuzzers
-for fuzzname in utils parse tokenize addr_parse uri request preq; do
+for fuzzname in utils parse tokenize addr_parse uri request preq new; do
   $CC $CFLAGS -c \
     -I$SRC/fuzz-headers/lang/c -I./include -I./os/unix \
     -I./srclib/apr/include -I./srclib/apr-util/include/ \
