--- Makefile.am.o	2023-09-15 19:16:17.303370323 +0000
+++ Makefile.am	2023-09-15 19:18:07.215873873 +0000
@@ -223,9 +223,11 @@
 endif
 
 if NEED_FUZZING
-check_PROGRAMS = fuzz/input-fuzzer
+check_PROGRAMS = fuzz/input-fuzzer fuzz/new-fuzzer
 fuzz_input_fuzzer_LDFLAGS = $(FUZZING_LIBS)
 fuzz_input_fuzzer_LDADD = $(LDADD) $(tmux_OBJECTS)
+fuzz_new_fuzzer_LDFLAGS = $(FUZZING_LIBS)
+fuzz_new_fuzzer_LDADD = $(LDADD) $(tmux_OBJECTS)
 endif
 
 # Install tmux.1 in the right format.
