--- build.sho	2025-02-11 20:04:14.715859770 +0000
+++ build.sh	2025-02-11 20:00:14.738664093 +0000
@@ -40,4 +40,11 @@
     $SRC/sqlite3/test/ossfuzz.o -o $OUT/ossfuzz \
     $LIB_FUZZING_ENGINE ./sqlite3.o
 
+$CC $CFLAGS -I. -c \
+    $SRC/sqlite3/test/newfuzzer.c -o $SRC/sqlite3/test/newfuzzer.o
+
+$CXX $CXXFLAGS \
+    $SRC/sqlite3/test/newfuzzer.o -o $OUT/newfuzzer \
+    $LIB_FUZZING_ENGINE ./sqlite3.o
+
 cp $SRC/*.options $SRC/*.dict $SRC/*.zip $OUT/
